backstop.json

```diff
@@ -26,57 +26,80 @@
     {
       "label": "Homepage",
       "url": "http://localhost:4000/",
-      "referenceUrl": "https://vulekamali.gov.za"
+      "referenceUrl": "https://vulekamali.gov.za",
+      "delay": 2000
```

Comment:
Just a tip - from my experience, using a delay in a test suggests your test is going to fail intermittently and cause a of frustration. A better approach is to block until the thing your test relies on is ready. We can chat about some reliable ways of doing this if you like. There's always a way. The delay might be a reasonable tradeoff right now to avoid writing a whole bunch of test code. Just be aware this can be a pain if things happen slower than expected for some reason.

Reply:
@jbothma Turns out it was a bug with the specific headless Chrome version I was using: 

![image](https://user-images.githubusercontent.com/14258328/40229830-bb6dc640-5a95-11e8-9568-504d9338a160.png)

Reply:
Will remove the delay.